### PR TITLE
Remove MBEDTLS_ENTROPY_NV_SEED and MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES

### PR DESCRIPTION
**Pull request by urutva**
Following changes are done,
1. Remove static private and public key and let attestation service
generate them
2. Remove MBEDTLS_ENTROPY_NV_SEED config option and relevant method
fake_set_initial_nvseed() used to inject fake NV seed
3. Remove MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES config option to utilize
Hardware TRNG available on K64F

Signed-off-by: Devaraj Ranganna <devaraj.ranganna@arm.com>

### CODE REVIEW

**Comment by urutva** [comment]
@Patater  Please review

**Comment by Patater** [comment]
First commit needs breaking up, as it tries to do too much all at once. Each line item in the commit description should be its own commit.

**Comment by urutva** [comment]
@Patater done.

File: mbed_app.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "target_overrides": {
         "*": {
              "platform.stdio-convert-newlines": true,
+             "platform.stdio-baud-rate": 115200,
```
Comment (jamesbeyond) [inline mbed_app.json]: can you remove this line, and keep this default as 9600

**Review by jamesbeyond — CHANGES_REQUESTED** [review]

**Comment by Patater** [comment]
Could you remove the `1.` from the commit title of `1. Remove static private and public key and let attestation service`?

**Comment by urutva** [comment]
@jamesbeyond @Patater done

**Review by jamesbeyond — APPROVED** [review]
LGTM

**Review by Patater — APPROVED** [review]

**Comment by 0xc0170** [comment]
We suspect this broke Mbed OS master testing, see internal ticket MBEDOSTEST-726

File: mbedtls_user_config.h [hunk 2]
```diff
@@ -25,8 +25,6 @@
 /* Enable the default implementation of the PSA entropy injection API if we are
  * building for an SPE. */
 #if defined(COMPONENT_PSA_SRV_IMPL) || defined(COMPONENT_PSA_SRV_EMUL)
-#   define MBEDTLS_ENTROPY_NV_SEED
-#   define MBEDTLS_NO_DEFAULT_ENTROPY_SOURCES
 #   define MBEDTLS_PSA_INJECT_ENTROPY
```
Comment (Patater) [inline mbedtls_user_config.h:28]: We should remove `MBEDTLS_PSA_INJECT_ENTROPY` as well

**Pull request merged**